docs/open/scholarship.md

```diff
@@ -0,0 +1,31 @@
+---
+id: open-scholarship
+title: Open Scholarship
+custom_edit_url: null
+---
+
+Open scholarship is a concept that extends the principles of openness beyond scientific research to encompass all scholarly activities, including teaching, learning, and public engagement. By embracing open practices, scholars can enhance the accessibility, transparency, and impact of their work, fostering a more inclusive and collaborative academic ecosystem.
+
+Saint Louis University is a member of [HELIOS Open](https://www.heliosopen.org/).
+
+## Key Principles of Open Scholarship
+
+* **Open Access:** Making scholarly publications, educational resources, and research outputs freely available online, without subscription barriers, to facilitate widespread dissemination and knowledge sharing.
+* **Open Data:** Sharing research data, along with appropriate metadata and documentation, to enable reuse, validation, and new discoveries across disciplines.
+* **Open Source:** Utilizing and contributing to open source software, tools, and platforms, fostering collaboration, transparency, and cost-effectiveness in scholarly practices.
+* **Open Peer Review:** Embracing transparent and inclusive peer review processes, where reviews are openly available, promoting accountability and constructive feedback.
+* **Open Pedagogy:** Encouraging the creation, adaptation, and sharing of open educational resources (OERs) and open pedagogical practices, fostering equity and innovation in teaching and learning.
+* **Open Collaboration:** Engaging in collaborative scholarly practices, fostering interdisciplinary collaborations, and embracing diverse perspectives to tackle complex challenges.
+* **Open Science:** Applying open science principles, such as preregistration, reproducible research, and open notebooks, to enhance the transparency and reproducibility of scientific research.
```

Comment:
What is preregistration?

Reply:
I found the answer later, when reviewing science.md. So there's definitely an overlap between Open Scholarship and Open Science (and other open things). Maybe an opportunity for a Venn diagram to visualize this overlap?